vnext/include/ReactWindowsCore/unicode.h

```diff
@@ -54,11 +54,11 @@ class UnicodeConversionException : public std::runtime_error
 //
 // For (2), utf8 must be null terminated. The behavior is undefined otherwise.
 //
-/* (1) */ std::wstring utf8ToUtf16(const char* utf8, size_t utf8Len);
-/* (2) */ std::wstring utf8ToUtf16(const char* utf8);
-/* (3) */ std::wstring utf8ToUtf16(const std::string& utf8);
+/* (1) */ std::wstring __stdcall utf8ToUtf16(const char* utf8, size_t utf8Len);
```

Comment:
At least some of these string conversion methods are being exported from react-native-win32.dll. I probably overlooked the respective file, but in case it's really missing: Please ensure this doesn't need to be accompanied by a .def change.

Reply:
It doesn't, for StdCall is the default convention.
Please not I have reverted that change anyways.